python/README.md

```diff
@@ -25,44 +40,46 @@ def main():
 
 if __name__ == "__main__":
     main()
-```
-
-## Development
-
-### Important Packages & Tools
-
-#### [`poetry`](https://python-poetry.org)
-
-Python dependency management. 
 
-ARFlow uses `poetry` to manage dependencies and run commands. Commands can be found in the `pyproject.toml` file in the `[tool.poetry.scripts]` section and can be run via `poetry run <command>`.
+```
 
-#### [`protobuf`](https://protobuf.dev)
+Save the above code to a file, e.g., `simple_server.py`, and run it:
 
-A language-neutral, platform-neutral, extensible mechanism for serializing structured data.
+```bash
+python3 simple_server.py
+```
 
-ARFlow uses `protobuf` to define the communication protocol between the server and the client. The protocol is defined in [`service.proto`](../protos/arflow/service.proto) and can be compiled using [`compile.sh`](../protos/scripts/compile.sh).
+Once you have your server running, you can start your ARFlow clients and connect them to the server. The server will start collecting data from the clients and save it to a `pickle` file at the end of the session.
 
-#### [`pickle`](https://docs.python.org/3/library/pickle.html)
+You can visualize the data using the ARFlow Player:
 
-Implements binary protocols for serializing and deserializing Python objects. Pickling is the same as serialization, marshalling, or flattening in other languages. The inverse operation is called unpickling.
+```python
+"""A simple example of replaying saved ARFlow data."""
 
-#### [`asyncio`](https://docs.python.org/3/library/asyncio.html)
+import arflow
+from .simple_server import CustomService
 
-A library to write **concurrent** code using using the `async` and `await` syntax. Perfect for writing IO-bound and high-level structured network code.
 
-#### [`rerun.io`](https://github.com/rerun-io/rerun)
+def main():
+    """Run the example."""
+    player = arflow.ARFlowPlayer(
+        CustomService, frame_data_path="frames_2024_09_06_18_21_17.pkl"
```

Comment:
I think we should replace this `frame_data_path` value to a placeholder, something like `FRAME_DATA_PATH.pkl`